Generate.java

```diff
@@ -904,10 +904,18 @@ String generate(boolean definition, boolean uniqueHandle, boolean customVectorAl
     static void generateModule(List<String> enums, List<String> structs, List<Handle> handles) throws IOException {
         Files.writeString(Path.of("src/vk_mem_alloc.cppm"), processTemplate("""
                 module;
+
+                #if defined( __cpp_lib_modules )
+                #define VMA_ENABLE_STD_MODULE
+                #endif
```

Comment:
No. It is intended to be specified during module compilation to enable use of VMA-hpp as a module without vulkan-hpp module. I do not know if such a mode of operation has any merit but it may be of some value

Reply:
> to enable use of VMA-hpp as a module without vulkan-hpp module?

As I can see, it's the other way - `vulkan.hpp` is included by default and one needs to explicitly define `VMA_ENABLE_VULKAN_HPP_MODULE` in order to use `vulkan_hpp` module instead. But when compiling VMA-Hpp as a module, it kinda makes more sense for it to import `vulkan_hpp` module by default.

Reply:
Ok, I will add a disable flag